lib/vdsm/supervdsm_api/udev.py

```diff
@@ -115,24 +116,32 @@ def rmAppropriateMultipathRules(thiefId):
 
 
 @expose
-def add_managed_udev_rule(vol_id, path):
-    rule_file = _UDEV_RULE_FILE_NAME_MANAGED % vol_id
-    symlink = os.path.relpath(path, '/dev/')
-    rule = 'SYMLINK=="%s", RUN+="%s %s:%s $env{DEVNAME}"\n' % (
-        symlink, EXT_CHOWN, DISKIMAGE_USER, DISKIMAGE_GROUP)
+def add_managed_udev_rule(sd_id, vol_id, path):
+    rule_file = _UDEV_RULE_FILE_NAME_MANAGED % (sd_id, vol_id)
+    device = os.path.relpath(path, '/dev/')
+    if os.path.islink(path):
+        rule = 'SYMLINK=="%s", RUN+="%s %s:%s $env{DEVNAME}"\n' % (
+            device, EXT_CHOWN, DISKIMAGE_USER, DISKIMAGE_GROUP)
+    else:
+        rule = 'KERNEL=="%s", RUN+="%s %s:%s $env{DEVNAME}"\n' % (
+            device, EXT_CHOWN, DISKIMAGE_USER, DISKIMAGE_GROUP)
     with open(rule_file, "w") as rf:
         _log.debug("Creating rule %s: %r", rule_file, rule)
         rf.write(rule)
 
 
 @expose
 def trigger_managed_udev_rule(path):
-    _udevTrigger(property_matches=(('DEVLINKS', path),))
+    # Example path: /dev/rbd/volumes/volume-d1530bb1-...-1adbcbc4eedc
```

Comment:
We need example for each case in the if block, best 3 example for /dev/mapper/xxx, /dev/rbd/volumes/volume-yyy and /dev/nvme...

Since we need the examples in both add_managed_udev_rules and trigger_managed_udev_rule - maybe add a comment in the module about managed volumes paths?